dev/tasks/bundle/buildclassiceditor.js

```diff
@@ -0,0 +1,60 @@
+/**
```

Comment:
We need a generator for such a file and/or make it possible to pass path to such a file to the builder. The generator would accept feature paths (BTW. see `PluginCollection.getPluginPath()`) and generate this file. Passing a path to the file will be useful for some integration cases when you have e.g. your own plugins in a totally different place.